soh/soh/Enhancements/custom-message/CustomMessageManager.cpp

```diff
@@ -367,6 +376,59 @@ static size_t NextLineLength(const std::string* textStr, const size_t lastNewlin
   }
 }
 
+
+size_t CustomMessage::FindNEWLINE(std::string& str, size_t lastNewline) const {
+    size_t newLine = str.find(NEWLINE()[0], lastNewline);
+    bool done;
+    do {
+        done = true;
+        if (newLine != 0){
+            switch (str[newLine - 1]){
+                case '\x05'://COLOR
+                case '\x06'://SHIFT
+                case '\x07'://TEXTID
+                case '\x0C'://BOX_BREAK_DELAYED
+                case '\x0E'://FADE
+                case '\x11'://FADE2
+                case '\x12'://SFX
+                case '\x13'://ITEM_ICON
+                case '\x14'://TEXT_SPEED
+                case '\x15'://BACKGROUND
+                case '\x1E'://POINTS/HIGH_SCORE
```

Comment:
Could these use the macros from message_data_fmt?

Reply:
There are name overlaps between CustomMessageManager's helper functions and some of the defines in message_data_fmt, causing build errors if I include it. It could be fixed with better src and CustommessageManager intergration, but it would be beyond the scope of this bugfix.